Tests/Unit/pjsip4net.Tests/Calls/given_a_call.cs

```diff
@@ -59,7 +75,15 @@ public void when_transfer_in_active_media_state__should_delegate_call_to_provide
             _sut.Transfer("sip:sip@sip");
 
             //assert
-            _provider.Verify(x => x.TransferCall(It.Is<int>(x1 => x1 == _sut.Id), It.Is<string>(x1 => x1 == "sip:sip@sip")), Times.Once());
+
+            if (shouldDelegate)
```

Comment:
That would compile down to the same thing for me ;-)
But if it floats your boat....